Cython/Utility/CommonStructures.c

```diff
@@ -1,6 +1,6 @@
 /////////////// FetchCommonType.proto ///////////////
 
-static PyTypeObject* __Pyx_FetchCommonType(PyTypeObject* type);
+static CYTHON_UNUSED PyTypeObject* __Pyx_FetchCommonType(PyTypeObject* type);
```

Comment:
Lovely. ;-)

Reply:
This is because I'm only requiring this from utilitycode but only `__Pyx_FetchSharedCythonABIModule`. On second look it'd probably be better splitting it into 2 bits so I can get `__Pyx_FetchSharedCythonABIModule` on its own.